docs/tools/offline/cli-device-management.md

```diff
@@ -0,0 +1,76 @@
+### Updating Devices with Mbed CLI
+
+Mbed CLI includes features that help prepare and ship updates for devices managed through Pelion. The Mbed CLI uses the subcommands starting with `mbed device-management`, `mbed dev-mgmt` or `mbed dm` to manage devices.
+
+Start by configuring your Mbed Cloud SDK API Key, target and toolchain. Obtain the API Key from the Pelion portal.
+
+```
+$ mbed config -G CLOUD_SDK_API_KEY ak_...
+$ mbed target K64F
+$ mbed toolchain GCC_ARM
+```
+
+Next, initialize the device management feature of Mbed CLI with the following command:
+
+```
+$ mbed device-management init -d "<company domain name>" -m "<product model identifier>"
+```
+
+First, this command asks for information about your update certificate. When Mbed CLI has enough information, it creates several files:
+* A certificate in `.update-certificates/default.der`.
+* A matching private key in `.update-certificates/default.key.pem`.
+* A set of default settings in `.manifest_tool.json`.
+* Mbed Cloud settings in `.mbed_cloud_config.json`.
+* Mbed Cloud developer credentials in `mbed_cloud_dev_credentials.c`
```

Comment:
So this is only for development? What about production flows?

Reply:
Since this is in the Mbed OS docs, it should (as part of the intro way up there around line 1) link to the Pelion docs. As an Mbed OS user, I've no idea what all these concepts mean.